assignments/prototypes.js

```diff
@@ -32,6 +44,13 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+Humanoid = (humanoidAttributes) => {
+  this.team = humanoidAttributes.team;
+  this.weapons = humanoidAttributes.weapons;
+  this.language = humanoidAttributes.language;
+  this.greet(`${GameObject.name} offers a greeting in ${this.language}`);
+  CharacterStats.call(this.statAttributes);
+}
```

Comment:
This defeats the purpos and is a work around. The issue is just in your inheritance. This should be:

```js
function Humanoid(attributes) {
	CharacterStats.call(this, attributes);
	this.team = attributes.team;
	this.weapons = attributes.weapons;
	this.language = attributes.language;
}

Humanoid.prototype = Object.create(CharacterStats.prototype);
Humanoid.prototype.greet = function() {
	return `${this.name} offers a greeting in ${this.language}`;
};
```